# Goal
float64 dummy
---
# Result
bool success
string message
---
# Feedback
float64 distance_ahead
float64 current_x
float64 current_y
